docker/filebeat/filebeat.yml

```diff
@@ -25,190 +25,142 @@ filebeat.autodiscover:
       templates:
         - condition:
             contains:
-              docker.container.image: "apm-server"
+              docker.container.name: "opbeans-"
```

Comment:
How many different `opbeans-*` container types do we have? As the logs coming from these containers are so different, I wonder if we should treat them differently.

Reply:
>How many different opbeans-* container types do we have? 

There are 7 different agents (.NET, Java, Go, Node.js, Python, Ruby, and RUM)

>As the logs coming from these containers are so different

We'd try that they aren't, but right now they are different

>I wonder if we should treat them differently.

Do you mean different configs or different indices?

Reply:
With the FB setup today, it will make your life much easier if we can get them into index. Because of this I'm thinking of different configs like you already did for json and non json to split it up even further. Like this if 5-6 work, you can move the data of the conflicting one for example under its own prefix.